libretroshare/src/pqi/authssl.h

```diff
@@ -42,109 +42,191 @@
 #include <map>
 
 #include "util/rsthreads.h"
+#include "util/rsdebug.h"
 
 #include "pqi/pqi_base.h"
 #include "pqi/pqinetwork.h"
 #include "pqi/p3cfgmgr.h"
 
-/* This #define removes Connection Manager references in AuthSSL.
- * They should not be here. What about Objects and orthogonality?
- * This code is also stopping immediate reconnections from working.
- */
+#include "retroshare/rsevents.h"
+
+
+#define RS_DEBUG_AUTHSSL 1
 
 class AuthSSL;
 
-class sslcert
+struct sslcert
 {
-        public:
-        sslcert(X509* x509, const RsPeerId& id);
-        sslcert();
+	sslcert(X509* x509, const RsPeerId& id);
 
-        /* certificate parameters */
-        RsPeerId id;
-        std::string name;
-        std::string location;
-        std::string org;
-        std::string email;
+	/* certificate parameters */
+	RsPeerId id;
+	std::string name;
+	std::string location;
+	std::string org;
 
-        RsPgpId issuer;
-        PGPFingerprintType fpr;
```

Comment:
Why would you like to remove the fingerprint? It's going to be useful. Please keep it for a while. After we switch to 160 bits IDs this is going to help.

Reply:
It was never used and was misleading to keep it there because it was suggesting that value was populated, while it was always 0. I think it is better to re-add it when we populate it not now.

Reply:
ok